docs/en/Clients-Hadoop.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: global
+title: Hadoop-Compatible Java Client
+nickname: Hadoop-Compatible Java
+group: Clients
+priority: 2
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio provides access to data through a filesystem interface. Files in Alluxio offer write-once
+semantics: they become immutable after they have been written in their entirety and cannot be read
+before being completed. Alluxio provides two different Filesystem APIs, a native API and a Hadoop
+compatible API. The native API provides better performance, while the Hadoop compatible API gives
```

Comment:
the performance of the native API is very similar, can we remove the performance part? Same in Alluxio-Clients-Java-Native